### ISSUE TITLE
Make sure CKAN harvesting from Salford CC portal to DGU works fine

### ISSUE THREAD

**Issue by jobarratt** [okfn/ckanext-salford#11]
We are concerned that if we activate harvesting on the data.gov site to take data from our portal as suggested we will end up duplicating content in data.gov, (as we are presuming that Sean did a one off import of our datasets  from data.gov) – please can you advise.

E.g. is this the case/or can you withdraw datasets from the harvest?

**Comment by jobarratt** [comment]
TL;DR: We need to update the ids of the datasets on the Salford site to match the ones in DGU

To handle duplicates across different sites the CKAN harvester uses the same id that the dataset has on the remote site (eg Salford) when creating it on the local site (eg DGU). If a dataset with the same id is harvested successively it will be updated on the local site instead of created.

This scenario works well when Datasets are harvested from Salford to DGU in the first place, as Salford is the canonical source of the metadata. But if I understand correctly, on our case datasets were imported the other way to populate the site. This was not done via harvesting but via a script included on the extension. This script didn’t take the remote ids into account and new ones were generated when creating the datasets, so right now if DGU harvested the Salford CKAN instances it would duplicate the records.

Another potential issue is that eventually the metadata flow needs to be Salford > DGU datasets need to be updated to the latest version and _only_ edited on the Salford site. For instance this dataset has 2 resources in DGU, and was edited there on 26th February. If harvesting was activated now, these updates would be lost.

https://salforddataquay.uk/dataset/land-and-property-assets-transparency-code
http://data.gov.uk/dataset/land-and-property-assets-transparency-code

https://salforddataquay.uk/api/action/package_show?id=land-and-property-assets-transparency-code
http://data.gov.uk/api/action/package_show?id=land-and-property-assets-transparency-code

**Comment by amercader** [comment]
```
+----------------+                +-----------------+
|                |                |                 |
|   Salford CC   +--------------> |   Data.gov.uk   |
|                |   CKAN         |                 |
+----------------+   harvesting   +--------+--------+
                                           |         
        ^                                  |         
        |                                  |         
        +----------------------------------+         
                   Initial import                    

```

Actions:
- [x] Modify `etl.py` and the custom dataset schema so imported dataset have the same id as in DGU
- [x] Clarify with the client what is the metadata flow (ie it needs to be updated on the Salford site)
- [x] Activate harvesting on DGU (?)
